roadmap/implementors-guide/guide.md

```diff
@@ -883,8 +883,67 @@ Included: Option<()>,
 
 ### The Validity Module
 
+After a backed candidate is made available, it is included and proceeds into an acceptance period during which validators are randomly selected to do secondary checks of the parablock. Any reports disputing the validity of the candidate will cause escalation, where even more validators are requested to check the block, and so on, until either the parablock is determined to be invalid or valid. Those on the wrong side of the dispute are slashed and, if the parablock is deemed invalid, the relay chain is rolled back to a point before that block was included.
```

Comment:
I write say "(secondary) approval checks" or similar, but avoid saying just "secondary checks" since someone might read secondary as less important.